docs/pages/application-access/reference.mdx

```diff
@@ -29,6 +29,9 @@ app_service:
   - name: "grafana"
     # Free-form application description.
     description: "This is an internal Grafana instance"
+    # Optional: For access to cloud provider APIs, specify the cloud provider.
+    # Allowed values are "AWS", "Azure", and "GCP".
+    cloud: ""
```

Comment:
I assume you would not typically set this for an app such as Grafana.

Should we expand the example to add separate `apps` entries for the cloud consoles with more realistic examples?